api/desecapi/tests/test_rrsets.py

```diff
@@ -364,9 +364,9 @@ def test_create_my_rr_sets_known_type_benign(self):
             'MX': ['10 example.com.', '20 1.1.1.1.'],
             'NAPTR': ['100  50  "s"  "z3950+I2L+I2C"     ""  _z3950._tcp.gatech.edu.'],
             'NS': ['ns1.example.com.'],
-            # 'OPENPGPKEY': [
-            #     'mG8EXtVIsRMFK4EEACIDAwQSZPNqE4tSxLFJYhX+uabSgMrhOqUizJhkLx82',  # key incomplete
-            # ],
+            'OPENPGPKEY': [
+                 'mG8EXtVIsRMFK4EEACIDAwQSZPNqE4tSxLFJYhX+uabSgMrhOqUizJhkLx82',  # key incomplete
```

Comment:
what does the comment in the end mean?

Reply:
it's not a valid key, but a truncated one (couldn't find one that fits the lmdb 500 byte limit)